drivers/spi/spi_ll_stm32.c

```diff
@@ -196,6 +196,11 @@ static void spi_stm32_complete(struct spi_stm32_data *data, SPI_TypeDef *spi,
 			/* NOP */
 		}
 	}
+	/* BSY flag is set when MODF is raised (but not only).
```

Comment:
"BSY flag is **cleared** when MODF is raised"
Also, put the comment end tag on the same line

Reply:
Done, thank you :)